kong/cmd/utils/prefix_handler.lua

```diff
@@ -253,6 +253,7 @@ local function compile_conf(kong_config, conf_template)
       end
 
       value = math.min(value, 16384)
+      value = math.max(value, 1024)
```

Comment:
We'd better show a warning to our users, and then they can adjust the `ulimit` (if needed)

Reply:
@ADD-SP there is already a warning of too low `ulimit`, but this is to make that Kong can at least start.